### PR TITLE
Created the donator thanks page for subscriptions

### PR DESCRIPTION
**Pull request by igorrfc**
**View:**
screen shot 2017-04-07 at 3 45 17 pm

### CODE REVIEW

File: config/locales/catarse_bootstrap/views/projects/subscriptions.en.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 en:
   projects:
     subscriptions:
+      show:
+        title: 'Donation succeed !'
```
Comment (marclerodrigues) [inline config/locales/catarse_bootstrap/views/projects/subscriptions.en.yml]: Do you really need this space at the end?

File: config/locales/catarse_bootstrap/views/projects/subscriptions.pt.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
 pt:
   projects:
     subscriptions:
+      show:
+        title: 'Doação realizada com sucesso !'
```
Comment (marclerodrigues) [inline config/locales/catarse_bootstrap/views/projects/subscriptions.pt.yml]: The same here.

File: spec/decorators/subscription_decorator_spec.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe SubscriptionDecorator do
+  include Rails.application.routes.url_helpers
```
Comment (marclerodrigues) [inline spec/decorators/subscription_decorator_spec.rb]: I believe you have access to url helpers using `h.` without needing to explicitly include the helpers.
So if you need to access 'projects_url' you can call `h.projects_url`.

Comment (igorrfc) [inline spec/decorators/subscription_decorator_spec.rb]: Great tip, @marclerodrigues. I didn't knew that. Thanks !

**Pull request merged**